pkg/cfn/manager/nodegroup.go

```diff
@@ -281,36 +170,43 @@ func (c *StackCollection) GetNodeGroupSummaries(name string) ([]*NodeGroupSummar
 	// Create an empty array here so that an object is returned rather than null
 	summaries := []*NodeGroupSummary{}
 	for _, s := range stacks {
-		ngPaths, err := getNodeGroupPaths(s.Tags)
+		nodeGroupType, err := GetNodeGroupType(s.Tags)
 		if err != nil {
 			return nil, err
 		}
 
-		summary, err := c.mapStackToNodeGroupSummary(s, ngPaths)
+		if nodeGroupType == api.NodeGroupTypeUnmanaged {
+			ngPaths, err := getNodeGroupPaths(s.Tags)
+			if err != nil {
+				return nil, err
+			}
 
-		if err != nil {
-			return nil, errors.Wrap(err, "mapping stack to nodegroup summary")
-		}
+			summary, err := c.mapStackToNodeGroupSummary(s, ngPaths)
 
-		asgName, err := c.GetAutoScalingGroupName(s)
-		if err != nil {
-			return nil, errors.Wrap(err, "getting autoscalinggroupname")
-		}
+			if err != nil {
+				return nil, errors.Wrap(err, "mapping stack to nodegroup summary")
+			}
 
-		summary.AutoScalingGroupName = asgName
+			asgName, err := c.getUnmanagedNodeGroupAutoScalingGroupName(s)
+			if err != nil {
+				return nil, errors.Wrap(err, "getting autoscalinggroupname")
+			}
 
-		asgDesiredCapacity, err := c.GetAutoScalingGroupDesiredCapacity(asgName)
-		if err != nil {
-			return nil, errors.Wrap(err, "getting autoscalinggroup desired capacity")
-		}
-		if asgDesiredCapacity != nil {
-			summary.DesiredCapacity = int(*asgDesiredCapacity)
-		}
+			summary.AutoScalingGroupName = asgName
 
-		if name == "" {
-			summaries = append(summaries, summary)
-		} else if summary.Name == name {
-			summaries = append(summaries, summary)
+			scalingGroup, err := c.GetAutoScalingGroupDesiredCapacity(asgName)
+			if err != nil {
+				return nil, errors.Wrap(err, "getting autoscalinggroup desired capacity")
+			}
+			summary.DesiredCapacity = int(*scalingGroup.DesiredCapacity)
+			summary.MinSize = int(*scalingGroup.MinSize)
+			summary.MaxSize = int(*scalingGroup.MaxSize)
+
+			if name == "" {
+				summaries = append(summaries, summary)
+			} else if summary.Name == name {
+				summaries = append(summaries, summary)
+			}
```

Comment:
This can be shortened to:
```suggestion
			if name == "" || summary.Name == name {
				summaries = append(summaries, summary)
			} 
```